README.md

```diff
@@ -60,6 +60,19 @@ fn main() {
 }
 ```
 
+Testing
+-------
+The unit tests are automatically generated based on upstream fixtures and should not be edited manually.
+
+    $ cargo test
+
+
+To generate the test cases, install `ChromeDriver` and the following Ruby Gems: `watir`, `watir-webdriver`. Run:
```

Comment:
Could you include a link to this page?

http://chromedriver.chromium.org

Reply:
Also a simple blurb about installing these tools would be helpful.

* Download the `chromedriver` binary and put it somewhere on your $PATH
* Linux/MacOS Example:
```bash
$ cp chromedriver /usr/local/bin
```
* Install the Ruby Gems:

```bash
$ sudo gem install watir watir-webdriver
```

* Generate the tests

```bash
$ ruby gentest/gentest.rb
```